app/fastlane/helpers/ios.rb

```
@@ -64,13 +66,13 @@
         latest = Fastlane::Actions::LatestTestflightBuildNumberAction.run(
           api_key: api_key,
           app_identifier: ENV["IOS_APP_IDENTIFIER"],
-          platform: "ios"
+          platform: "ios",
```

RuboCop (1.76.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)